### DISCUSSION TITLE
Response Curve for a channel doesn't match the ROI chart

### DISCUSSION THREAD

**Discussion by openhubber** [google/lightweight_mmm#247] [Q&A]
Response Curve:
![Screenshot 2023-09-12 at 10 37 27 AM](https://github.com/google/lightweight_mmm/assets/36459794/5369bc91-3311-48fa-8f02-5a607e1695b8)

I built a MMM model with the click data from different marketing channels and attached please find the response curve for one channel. The X axis is the daily spend, and we know around 70% of the daily spend is less than 100,000 and the average spend is $90,000. Based on the response curve, we can see the two estimated slope curves. We also observed from the ROI chart (output of lightweightMMM) that the ROI for this channel is 2.3. But the response curve doesn't seem to give 2.3 as an average ROI. Any thoughts on this?

**Comment by DiddyDaw** [comment]
Is the curve inclusive of carryover (adstock) or not. That would significantly change the story in the mid-term.

**Comment by openhubber** [comment]
Thanks DiddyDaw. I use the "hill_adstock" so I think the carryover effect is included in the model. 

But I am not sure if the response curve considered the carryover effect: if the response curve is only about each day's revenue vs the same day's spend, then carryover effect is not considered.
